lib/gemstash/cli/start.rb

```diff
@@ -11,46 +11,28 @@ class Start < Gemstash::CLI::Base
       def run
         prepare
         setup_logging
-        store_daemonized
         @cli.say("Starting gemstash!", :green)
         Puma::CLI.new(args, Gemstash::Logging::StreamLogger.puma_events).run
       end
 
     private
 
       def setup_logging
-        return unless daemonize?
-
+        # Enables logging at all times, perhaps there needs to be a new option for this?
```

Comment:
This code comment is about "Does the CLI need an option like `--run-in-foreground` to not redirect the log output into a file?"

Reply:
Seems like this uses "logger in a signal handler"

Compare this comment, and see how it refers to a nice closure trick from Sidekiq:
https://github.com/resque/resque/issues/1493#issuecomment-238481429

By removing code which depended on the `daemonize?`, I got the expected working behaviour.